src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -405,6 +405,21 @@ void esbmc_parseoptionst::get_command_line_options(optionst &options)
   }
 #endif
 
+  // parallel solving activates "--multi-property"
+  if(cmdline.isset("parallel-solving"))
+  {
+    options.set_option("result-only", true);
+    options.set_option("base-case", true);
```

Comment:
Does this break incremental strategies?

Reply:
I believe it will not break the incremental strategies. 
The idea is that previously we need the combined option `--base-case --multi-property --parallel-solving` to enable parallel mode. Now we only need to enable `--parallel-solving`

Reply:
I suggest testing this on sv-comp to be sure.